docs/RELEASE_NOTES.md

```diff
@@ -1,17 +1,13 @@
-# MongoDB Kubernetes Operator 0.6.2
+# MongoDB Kubernetes Operator 0.6.3
 
 ## Kubernetes Operator
 
 - Changes
-  - stability improvements when changing version of MongoDB.
-  - increased number of concurrent resources the operator can act on.
-  - mongodb will now send its log to stdout by default.
-  - changed the default values for `MONGODB_REPO_URL` and `MONGODB_IMAGE` in the operator deployment
-  - added the support of arbiters for the replica sets.
+  - added the support for arbiters for the replica sets.
```

Comment:
[nit] Mind rewording this phrase? it seems a bit difficult to understand. Maybe something like:

- "Members of a Replica Set can be configured as arbitrers"